x/evm/types/legacy/testutil/helper.go

```diff
@@ -0,0 +1,96 @@
+package testutil
+
+import (
+	"testing"
+
+	"github.com/evmos/ethermint/x/evm/types"
+	legacytypes "github.com/evmos/ethermint/x/evm/types/legacy"
+	"github.com/stretchr/testify/require"
+)
+
+func AssertParamsEqual(t *testing.T, legacyParams legacytypes.LegacyParams, params types.Params) {
```

Comment:
could this be replaced with `assert.Exactly`? or are there unchecked parameters that would break that?

Reply:
I don't think so since they are different types and not all values have the same name/identifier

Reply:
Tested it out, indeed does fail since they are different types